playbooks/check-nodepool-periodic.yaml

```diff
@@ -0,0 +1,65 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+---
+- name: Install hub and openlabcmd in nodepool
+  hosts: nodepool-launcher:!*-slave
+  become: yes
+  tasks:
+    - name: Install hub
+      shell:
+        cmd: |
+          ## check hub installed or not
+          if hub version >> /dev/null;then
+              echo "hub has been installed!"
+          else
+              echo "hub installing..."
+              wget https://github.com/github/hub/releases/download/v2.10.0/hub-linux-amd64-2.10.0.tgz
+              sudo tar -C /usr/local/src -xzf hub-linux-amd64-2.10.0.tgz
+              cd /usr/local/src/hub-linux-amd64-2.10.0
+              sudo ./install
+              hub version
+              echo "hub installed success!"
+          fi
+        executable: /bin/bash
+
+    - name: Install openlabcmd
+      pip:
+        name: openlabcmd
+        executable: pip3
+
+- name: Teardown labcheck crontab tasks for nodepool and zuul slave
+  hosts: nodepool-launcher-slave:zuul-scheduler-slave
+  become: yes
+  tasks:
+    - name: Teardown crontab tasks if exists(when slave master switched)
+      cron:
+        name: "{{ item }}"
```

Comment:
If there is only one cron entity, we can put the name here directly, but not blocker.

Reply:
done